docs/application/dotnet/guides/app-management/app-controls.md

```diff
@@ -145,6 +145,7 @@ appcontrol.ApplicationId = "org.tizen.TestApp";
 
 AppControl.SendLaunchRequest(appcontrol);
 ```
+To launch an AppControl asynchronously, you can use SendLaunchRequestAsync().
```

Comment:
@hjhun
It is okay although a developer don't use SendLaunchRequestAsync() method. When an asynchronous method is needed, he or she can use it. 

I think the following sentence is good enough:
"If you want to launch an AppControl asynchronously, use SendLaunchRequestAsync()."

Reply:
@journey2w 
I think so. I've updated the patchset.

Reply:
@journey2w,
I understand your point. Thank you for correcting me. I will take care of this ahead.